autowisp/diagnostics/get_from_lc.py

```diff
@@ -438,6 +439,8 @@ def main():
     """Avoid polluting global scope."""
 
     configuration = vars(parse_command_line())
+    if not configuration.get("project-home"):
```

Comment:
Argument will have `_` not `-`

The case when argument is defined should be handled

The argument should be added to command line parser

To make code easier to read have the `if` without `not` (in which case use default app data directory) and have this happen on `else`